packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -304,7 +296,7 @@ contract LongShortPair is Testable, Lockable {
         return IdentifierWhitelistInterface(finder.getImplementationAddress(OracleInterfaces.IdentifierWhitelist));
     }
 
-    function _getAddressWhitelist() internal view returns (AddressWhitelistInterface) {
```

Comment:
OZ comments on naming:

The [`_getAddressWhitelist` function](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol#L299) of the `LongShortPair` contract could be renamed to `_getCollateralWhitelist`.